### PR TITLE
Rich meta tags & snippets

### PR DESCRIPTION
**Pull request by maxkfranz**
Example for https://biofactoid.org/document/e1e4b079-258d-4258-b5a2-3a665046d880:

```
<meta name="keywords" content="biofactoid, app, bio, biology, paper, academic, academia, publish, peer, review, data, summary, digital, profile, connect, connected" />
<meta name="robots" content="all" />


<title>p27 controls Ragulator and mTOR activity in amino acid-deprived cells to regulate the autophagy-lysosomal pathway and coordinate cell cycle and cell growth. : Biofactoid</title>
<meta name="description" content="p27Kip1 inhibits LAMTOR1 via binding. : p27 controls Ragulator and mTOR activity in amino acid-deprived cells to regulate the autophagy-lysosomal pathway and coordinate cell cycle and cell growth." />
<meta name="author" content="Ada Nowosad, Pauline Jeannot, Caroline Callot, Justine Creff, Renaud Thierry Perchey, Carine Joffre, Patrice Codogno, Stephane Manenti, Arnaud Besson">
<script type="application/ld+json">
{
  "@context" : "http://schema.org",
  "@type" : "Article",
  "name" : "p27 controls Ragulator and mTOR activity in amino acid-deprived cells to regulate the autophagy-lysosomal pathway and coordinate cell cycle and cell growth.",
  "author" : [{ "@type": "Person", "name": Ada Nowosad }, { "@type": "Person", "name": Pauline Jeannot }, { "@type": "Person", "name": Caroline Callot }, { "@type": "Person", "name": Justine Creff }, { "@type": "Person", "name": Renaud Thierry Perchey }, { "@type": "Person", "name": Carine Joffre }, { "@type": "Person", "name": Patrice Codogno }, { "@type": "Person", "name": Stephane Manenti }, { "@type": "Person", "name": Arnaud Besson }],
  "datePublished" : "2020-10-01",
  "image" : "https://biofactoid.org/api/document/30673533-656d-4c78-bca3-ad155d3a0f4e.png",
  "articleBody" : "Autophagy is a catabolic process whereby cytoplasmic components are degraded within lysosomes, allowing cells to maintain energy homeostasis during nutrient depletion. Several studies reported that the CDK inhibitor p27Kip1 promotes starvation-induced autophagy by an unknown mechanism. Here we find that p27 controls autophagy via an mTORC1-dependent mechanism in amino acid-deprived cells. During prolonged starvation, a fraction of p27 is recruited to lysosomes, where it interacts with LAMTOR1, a component of the Ragulator complex required for mTORC1 activation. Binding of p27 to LAMTOR1 prevents Ragulator assembly and mTORC1 activation, promoting autophagy. Conversely, p27-/- cells exhibit elevated mTORC1 signalling as well as impaired lysosomal activity and autophagy. This is associated with cytoplasmic sequestration of TFEB, preventing induction of the lysosomal genes required for lysosome function. LAMTOR1 silencing or mTOR inhibition restores autophagy and induces apoptosis in p27-/- cells. Together, these results reveal a direct coordinated regulation between the cell cycle and cell growth machineries.",
  "url" : "https://biofactoid.org/document/30673533-656d-4c78-bca3-ad155d3a0f4e"
}
</script>
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@biofactoid">
<meta name="twitter:creator" content="@biofactoid">
<meta name="twitter:title" content="p27 controls Ragulator and mTOR activity in amino acid-deprived cells to regulate the autophagy-lysosomal pathway and coordinate cell cycle and cell growth.">
<meta name="twitter:description" content="Nat. Cell Biol. 22 (2020)">
<meta name="twitter:image" content="https://biofactoid.org/api/document/30673533-656d-4c78-bca3-ad155d3a0f4e.png">

```

Refs: Improve visibility via search (Google) #842

### CODE REVIEW

**Comment by maxkfranz** [comment]
Note: This also updates the binding text as it was always 'A binds with B' even if there was an arrow.

Now:

- A activates B via binding
- B inhibits B via binding
- A binds with B

**Comment by maxkfranz** [comment]
@jvwong, let's plan to merge in these PRs today.  #846 and this PR should close #842.

**Comment by maxkfranz** [comment]
Tags for homepage:

```
<meta name="keywords" content="biofactoid, app, bio, biology, paper, academic, academia, publish, peer, review, data, summary, digital, profile, connect, connected" />
<meta name="robots" content="all" />


<title>Biofactoid : Create a digital profile of your article's scientific discoveries and connect it to related biological research</title>
<meta name="description" content="Biofactoid creates a digital profile of scientific discoveries in an article and connects it to related research." />
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@biofactoid">
<meta name="twitter:creator" content="@biofactoid">
<meta name="twitter:title" content="Biofactoid">
<meta name="twitter:description" content="Create a digital profile of your article's scientific discoveries and connect it to related biological research.">
<meta name="twitter:image" content="https://biofactoid.org/image/card-banner.jpg">

```

Note: The card image for twitter should be updated: https://github.com/PathwayCommons/factoid/blob/89167d817ce5f3ed868e1cd37e371b79aae5da1d/src/styles/image/card-banner.jpg

File: src/views/index.html.ejs
Comment (jvwong) [inline src/views/index.html.ejs:18]: Google says use `headline` but maybe that's for news article https://developers.google.com/search/docs/data-types/article#technical-guidelines

**Review by jvwong — APPROVED** [review]

**Comment by jvwong** [comment]
Lemme find that banner

**Comment by maxkfranz** [comment]
Let's test with this before merging: https://search.google.com/test/rich-results

**Comment by maxkfranz** [comment]
OK, looks good: https://search.google.com/test/rich-results?id=rSu4igtQWtZ46Qi3TDO2Lw

File: src/views/index.html.ejs
Comment (maxkfranz) [reply src/views/index.html.ejs:18]: We have both now

Comment (jvwong) [reply src/views/index.html.ejs:18]: ok

**Comment by maxkfranz** [comment]
OK, let's merge this in once the banner is swapped out.  We can test unstable with the checker tool tomorrow: https://search.google.com/test/rich-results

**Comment by jvwong** [comment]
Preference?

**Comment by maxkfranz** [comment]
The second one may work better with the required aspect ratio:

> A URL to a unique image representing the content of the page. You should not use a generic image such as your website logo, author photo, or other image that spans multiple pages. Images for this Card support an aspect ratio of 2:1 with minimum dimensions of 300x157 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

See https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image

**Comment by jvwong** [comment]
Rich results OK

https://search.google.com/test/rich-results?id=VlUGVEkUUxB8qHZx8Kx0UA

**Comment by jvwong** [comment]
Twitter card ok

image

**Comment by jvwong** [comment]
Apparently you need to update or clear the cache for Twitter cards via https://cards-dev.twitter.com/validator

**Comment by maxkfranz** [comment]
Only if a page has been shared before. So the homepage needs a manual refresh, but articles should be ok.

**Pull request merged**
